plugins/hiscore/hiscore.dat

```diff
@@ -18866,3 +18866,11 @@ jackrabts:
 @:maincpu,program,60fe,1,ff,ff
 
 
+
+;
+; Color Computer (MESS)
```

Comment:
"MESS" no longer exists, and I'd put the full name "Tandy/Radio Shack Color Computer".  In the future we may want to split hiscore.dat by system but I think this works for now.  I do love including a hiscore.dat entry with the software list addition though. 👍